components/devtools_traits/Cargo.toml

```diff
@@ -20,3 +20,4 @@ msg = {path = "../msg"}
 serde = "1.0"
 servo_url = {path = "../url"}
 time = "0.1"
+uuid = {version = "0.8", features = ["v4", "serde"]}
```

Comment:
Is `"serde"` necessary?

Reply:
Since we need to implement Serialize and Deserialize traits on struct `WorkerId`, serde will be required.